### PR TITLE
compatibility with terra; new notebooks on optimization;

### PR DESCRIPTION
**Pull request by liupibm**
The following changes are made. 
1 updates the following notebooks to make them compatible with terra0.7 and adds the programming way for instance construction:
- community/aqua/artificial_intelligence/qsvm_kernel_directly.ipynb
- community/aqua/artificial_intelligence/qsvm_kernel_multiclass.ipynb
- community/aqua/artificial_intelligence/qsvm_variational.ipynb
- community/aqua/artificial_intelligence/svm_classical.ipynb
- community/aqua/artificial_intelligence/svm_classical_multiclass.ipynb
- community/aqua/optimization/clique.ipynb
- community/aqua/optimization/maxcut.ipynb
- community/aqua/optimization/stableset.ipynb

2 Added the following new notebooks:
- community/aqua/optimization/exact_cover.ipynb
- community/aqua/optimization/graph_partition.ipynb
- community/aqua/optimization/partition.ipynb
- community/aqua/optimization/set_packing.ipynb
- community/aqua/optimization/vertex_cover.ipynb

@attp Could you assign @chunfuchen as the reviewer. Thanks.

### CODE REVIEW

**Comment by chunfuchen** [comment]
@attp do you mind assigning me as a reviewer?

**Comment by liupibm** [comment]
@jaygambetta could you assign @chunfuchen as a reviewer. thanks.

**Comment by liupibm** [comment]
just a reminder: aqua is going to rename QSVM_Kernel as QSVMKernel.

**Comment by liupibm** [comment]
@chunfuchen aer backend updated for the notebooks.

File: community/aqua/artificial_intelligence/qsvm_variational.ipynb [hunk 1]
```diff
@@ -137,24 +159,105 @@
...
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "backend = get_aer_backend('qasm_simulator')\n",
+    "optimizer = SPSA(max_trials=10, c0=4.0, skip_calibration=True)\n",
```
Comment (chunfuchen) [inline community/aqua/artificial_intelligence/qsvm_variational.ipynb]: could we reproduce the result with only 10 iterations?

**Comment by chunfuchen** [comment]
1. The import statements should be simplified. For example, for the algorithms, you can do `from qiskit_aqua.algorithms import QSVMKernel`.
2. Please clean up the redundant imports
3. please remove `get_aer_backend`, either use `LegacySimulator` or `Aer`
4. for the notebook with CPLEX, please install CPLEX and re-run them
5. for the word `part` in each section, do you mind changing it to `Part`.
6. and other comments interleaved with the codes

File: community/aqua/artificial_intelligence/qsvm_kernel_directly.ipynb [hunk 2]
```diff
@@ -10,7 +10,15 @@
     "\n",
     "Please refer to [this file](./svm_qkernel.ipynb) for introduction.\n",
```
[the remark this answers is not in the document]
Comment (liupibm) [reply community/aqua/artificial_intelligence/qsvm_kernel_directly.ipynb]: fixed

**Comment by jaygambetta** [comment]
@liupibm and @chunfuchen when you guys are good can you push to master

**Comment by liupibm** [comment]
@jaygambetta @chunfuchen still need around 20 mins (for rerunning everything with the new Aer backend).

File: community/aqua/artificial_intelligence/qsvm_variational.ipynb (hunk 1 above)
Comment (liupibm) [reply community/aqua/artificial_intelligence/qsvm_variational.ipynb]: yes, this is what we observed with 10 trials.

**Comment by liupibm** [comment]
@jaygambetta @chunfuchen created another PR #457 for pushing changes to master.

**Pull request closed without merging**